<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGroup</class>
 <widget class="QWidget" name="CreateGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="group_avatar">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Icons/Resource/Icon/ic_group_avatar.png</normaloff>:/Icons/Resource/Icon/ic_group_avatar.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>62</width>
        <height>62</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="g_name">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-bottom-color: rgb(85, 85, 255);
border-style: outset;
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Group name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="g_description">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-bottom-color: rgb(85, 85, 255);
border-style: outset;
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Group description</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>351</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="close_groupW">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: rgb(126, 126, 126); background: white;}
QPushButton:hover{background: #f6f4ff; border:none;}</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="create_group">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{color: rgb(65, 109, 211); background: white;}
QPushButton:hover{background: #f6f4ff; border:none;}</string>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
